components/expenses/ExpensePayeeDetails.js

```diff
@@ -62,6 +62,18 @@ const ExpensePayeeDetails = ({ expense, host, isLoading, borderless, isLoadingLo
   const isCharge = expense?.type === expenseTypes.CHARGE;
   const isPaid = expense?.status === expenseStatus.PAID;
 
+  /*
+   * Displays the payee name as Legal name (Display name) if legal name exists.
+   * Example: Sudharaka (Suds)
+   */
+  const renderPayeeDisplayNameString = (legalName, payeeName) => {
```

Comment:
The function name is not correct, since we use it with something else than a payee below:

```es6
renderPayeeDisplayNameString(host.legalName, host.name) // Host is not a payee there
```

We'll need to re-use this in various places, we should either create a generic function in `lib/collective.lib.js` or a dedicated component. I've done the latest in the PDF service: https://github.com/opencollective/opencollective-pdf/blob/b119d3f0f00bbfe93528b95a593ee5c5443a234f/components/AccountName.js.

Reply:
Thanks. I've renamed the function to `displayPayeeNameWithLegalName`. Hope that sounds better? 🤔

Reply:
> The function name is not correct, since we use it with something else than a payee

The problem is with the word `payee`. When you do:
```es6
displayPayeeNameWithLegalName(host.legalName, host.name);
```

It looks wrong because host is not a payee, it's an account.

Reply:
Ops, I missed the fact the problem is with the word payee. I've renamed it to `concatenateDisplayNameWithLegalName`. How does that sound?